armor-read/src/main/java/com/rapid7/armor/store/S3ReadStore.java

```diff
@@ -246,7 +244,7 @@ public TableMetadata getTableMetadata(String tenant, String table) {
         allPrefixes.addAll(result.getCommonPrefixes().stream().map(o -> o.replace(Constants.STORE_DELIMETER, "")).collect(Collectors.toList()));
         lor.setContinuationToken(result.getNextContinuationToken());
       } while (result.isTruncated());
-      return allPrefixes.stream().filter(t -> !t.startsWith(TENANT_EXCLUDE_FILTER_PREFIX)).collect(Collectors.toList());
+      return allPrefixes.stream().filter(t -> !t.startsWith(StoreConstants.TENANT_EXCLUDE_FILTER_PREFIX)).collect(Collectors.toList());
```

Comment:
If they do a non-cache use option...lets populate the s3 entries like we did for the write side.